public/app/core/components/Select/OrgPicker.tsx

```diff
@@ -54,18 +54,16 @@ export class OrgPicker extends PureComponent<Props, State> {
     const { isLoading } = this.state;
 
     return (
-      <div className="org-picker">
-        <AsyncSelect
-          className={className}
-          isLoading={isLoading}
-          defaultOptions={true}
-          isSearchable={false}
-          loadOptions={this.getOrgOptions}
-          onChange={onSelected}
-          placeholder="Select organization"
-          noOptionsMessage={() => 'No organizations found'}
-        />
-      </div>
+      <Forms.AsyncSelect
+        className={className}
+        isLoading={isLoading}
+        defaultOptions={true}
+        isSearchable={false}
+        loadOptions={this.getOrgOptions}
+        onChange={onSelected}
+        placeholder="Select organization"
+        noOptionsMessage="No organizations found"
```

Comment:
nit: this was a function before, what is the correct prop? Or do we support both?

Reply:
With the new Select it is supposed to be a string. No reason why it was a function before.